### DISCUSSION TITLE
Grey out More troubleshooting information... when BlockAboutSupport=True

### DISCUSSION THREAD

**Discussion by MCHuijgen** [mozilla/policy-templates#949] [Ideas]
Hi, 

This is not so much about the policy templates itself, but an effect of a specific policy on the GUI of Firefox. 

I noticed that when setting the policy BlockAboutSupport to Enabled, the menu item Help > More troubleshooting information... is not being greyed out. I found that odd, since it opens about:support, which is blocked, as you would expect. A user in our organisation found this confusing and notified me as the sysadmin, and I have to agree with her. 

Is it possible to grey out the item More troubleshooting information... when the policy BlockAboutSupport is set to True?

**Comment by mkaply** [comment]
Yes, that seems reasonable :)

Do you mind opening a bugzilla bug for this?

https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Enterprise+Policies

IF not, I can take care of it.

**Reply by MCHuijgen** [reply]
Thanks, Mike! I created https://bugzilla.mozilla.org/show_bug.cgi?id=1792205 for this.

**Comment by mkaply** [comment]
This was fixed by that bug.
